pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.2.0.RELEASE</version>
+        <version>2.3.1.RELEASE</version>
```

Comment:
why are we upgrading SpringBoot?

Reply:
to bring it in line with the services repo